# 02-dfl-error-dir.patch by Luigi Gangitano <luigi@debian.org>
#
# Default error dir is under /usr/share/squid for Debian

--- src/defines.h	2002-08-08 22:17:39.000000000 +0200
+++ src/defines.h	2002-10-10 19:57:22.000000000 +0200
@@ -219,7 +219,7 @@
 
 /* were to look for errors if config path fails */
 #ifndef DEFAULT_SQUID_ERROR_DIR
-#define DEFAULT_SQUID_ERROR_DIR "/usr/local/squid/etc/errors"
+#define DEFAULT_SQUID_ERROR_DIR "/usr/share/squid/errors/English"
 #endif
 
 /* gb_type operations */
